packages/bot-web-ui/src/stores/toolbar-store.js

```diff
@@ -84,7 +84,7 @@ export default class ToolbarStore {
     }
 
     @action.bound
-    onResetCancelButtonClick() {
+    toggleResetDialog() {
         this.is_dialog_open = false;
     }
```

Comment:
The name isn't appropriate as a _toggle_ suggests the value of the boolean will be flipped.